src/libraries/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Windows/PkcsPalWindows.cs

```diff
@@ -177,6 +179,8 @@ public sealed override byte[] GetSubjectKeyIdentifier(X509Certificate2 certifica
                     return (T)(object)new RSACryptoServiceProvider(cspParams);
                 if (typeof(T) == typeof(DSA))
                     return (T)(object)new DSACryptoServiceProvider(cspParams);
+                if (typeof(T) == typeof(SlhDsa))
```

Comment:
I don't think this `if` is ever going to be hit because this is the CAPI path and I would be... surprised... if and PQC types worked with CAPI. `ECDsa` doesn't, either, which is why ECDsa is not here.